components/monitoring/README.md

```diff
@@ -1,6 +1,6 @@
-## On boarding to services to Monitoring
+## Onboarding to services to Monitoring
 
-### 1. Metrics exporter services
```

Comment:
If removing this, maybe you also need to remove `#### 2. Service monitors` below